server/qaRoutes.js

```diff
@@ -0,0 +1,41 @@
+const express = require('express');
+const router = express.Router();
+const axios = require('axios');
+const token = require('../client/config.js');
+const Promise = require('bluebird');
+
+
+var AtelierAPI = (method, endpoint, params = null, data = null, cb) => {
+  axios({
+    method: method,
+    url: endpoint,
+    baseURL: 'https://app-hrsei-api.herokuapp.com/api/fec2/hr-rfp/',
+    timeout: 1000,
+    params: params,
+    data: data,
+    headers: { Authorization: token.TOKEN},
+  })
+    .then(response => {
+      cb(null, response);
+    })
+    .catch(error => {
+      cb(error, null);
+    });
+};
+
+var AtelierAPIAsync = Promise.promisify(AtelierAPI);
```

Comment:
What is the purpose of this line? You don't call on AtelierAPIAsync. axios is a promise-based client, you can just return the function that you're calling inside AtelierAPI to return a promise object. Also, it's not best practice to combine promise async with callback async; I think your code will work just fine, but since you're using axios with promises, you should continue to use promises for response and error handling.

Reply:
This line is not used. I removed it. It makes sense to remove the callback function included, as Promise from axios itself should work. I removed callback in next PR.